services/QuillLMS/client/app/bundles/Evidence/helpers/containerRenderHelpers.tsx

```diff
@@ -99,42 +99,34 @@ export const renderPromptSteps = ({
   completedSteps,
   doneHighlighting,
   showReadTheDirectionsModal,
-  stepsHash,
   reportAProblem,
 }) => {
   const { currentActivity, } = activities
   const { submittedResponses, hasReceivedData, } = session
+
   if (!currentActivity || !hasReceivedData) return
 
-  // sort by conjunctions in alphabetical order: because, but, so
-  const steps =  orderedSteps(activities).map((prompt, i) => {
-    // using i + 2 because the READ_PASSAGE_STEP is 1, so the first item in the set of prompts will always be 2
-    const stepNumber = i + 2
-    const canBeClicked = completedSteps.includes(stepNumber - 1) || completedSteps.includes(stepNumber) // can click on completed steps or the one after the last completed
+  const stepNumber = activeStep - 2;
+  const prompts = orderedSteps(activities);
+  const prompt = prompts[stepNumber];
 
-    return (<PromptStep
+  return (<div className="prompt-steps">
+    {renderDirectionsSectionAndModal({ className: 'hide-on-mobile', closeReadTheDirectionsModal, activeStep, doneHighlighting, showReadTheDirectionsModal, activities })}
+    <PromptStep
       activateStep={activateStep}
-      active={stepNumber === activeStep}
       activityIsComplete={activityIsComplete}
-      canBeClicked={canBeClicked}
-      className={`step ${canBeClicked ? 'clickable' : ''} ${activeStep === stepNumber ? 'active' : ''}`}
+      className="step active"
       completeStep={completeStep}
       completionButtonCallback={completionButtonCallback}
-      everyOtherStepCompleted={everyOtherStepCompleted(stepNumber, completedSteps)}
-      key={stepNumber}
-      passedRef={stepsHash[`step${stepNumber}`]} // eslint-disable-line react/jsx-no-bind
+      everyOtherStepCompleted={everyOtherStepCompleted(activeStep, completedSteps)}
+      key={activeStep}
       prompt={prompt}
       reportAProblem={reportAProblem}
-      stepNumber={stepNumber}
-      stepNumberComponent={renderStepNumber(stepNumber, activeStep, completedSteps)}
+      stepNumber={activeStep}
+      stepNumberComponent={renderStepNumber(activeStep, activeStep, completedSteps)}
```

Comment:
Why are we passing the same variable twice here? Maybe in the method `renderStepNumber` we can name the argument `number` something more informative, so it's less confusing (I'm sure there's a good reason why but just looking at the method it's not clear from the variable names what `number` represents).

Reply:
good catch, I actually meant to delete this prop as we no longer need it